lib/cc/engine/analyzers/analyzer_base.rb

```diff
@@ -0,0 +1,43 @@
+module CC
+  module Engine
+    module Analyzers
+      class AnalyzerBase
```

Comment:
I think it might be a bit more idiomatic if this were named `CC::Engine::Analyzers::Base`. What do you think?